relay-general/src/store/transactions.rs

```diff
@@ -674,10 +682,10 @@ mod tests {
     }
 
     #[test]
-    fn test_discards_transaction_event_with_span_with_missing_op() {
+    fn test_defaults_transaction_event_with_span_with_missing_op() {
         let mut event = Annotated::new(Event {
             ty: Annotated::new(EventType::Transaction),
-            timestamp: Annotated::new(Utc.ymd(2000, 1, 1).and_hms(0, 0, 0)),
+            timestamp: Annotated::new(Utc.ymd(2000, 1, 1).and_hms(0, 0, 10)),
```

Comment:
Same consideration as above, making the test data more realistic: non-zero duration.